mysql-test/suite/innodb_gis/t/rtree.test

```diff
@@ -851,3 +851,21 @@ INSERT INTO `address` VALUES (1,'47 MySakila Drive',NULL,'Alberta',300,'','',0x0
 COMMIT;
 
 DROP TABLE address;
+
+--echo #
+--echo # MDEV-35116 InnoDB fails to set error index
+--echo #        for HA_ERR_NULL_IN_SPATIAL
+--echo #
+call mtr.add_suppression("BINLOG_BASE64_EVENT: Could not execute Write_rows_v1 event on table .*");
+
+BINLOG ' SOgWTg8BAAAAbgAAAHIAAAAAAAQANS42LjMtbTUtZGVidWctbG9nAAAAAAAAAAAAAAAAAAAAAAAA AAAAAAAAAAAAAAAAAABI6BZOEzgNAAgAEgAEBAQEEgAAVgAEGggAAAAICAgCAAAAAAVAYI8=';
```

Comment:
This statement could be moved right before the failing `BINLOG` statement, that is, after the failed attempt to create and populate the temporary table `t2`.